system_tests/timeboost_test.go

```diff
@@ -40,6 +40,114 @@ import (
 	"github.com/offchainlabs/nitro/util/stopwaiter"
 )
 
+func TestExpressLaneControlTransfer(t *testing.T) {
+	t.Parallel()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	tmpDir, err := os.MkdirTemp("", "*")
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		require.NoError(t, os.RemoveAll(tmpDir))
+	})
+	jwtSecretPath := filepath.Join(tmpDir, "sequencer.jwt")
+
+	seq, seqClient, seqInfo, auctionContractAddr, aliceBidderClient, bobBidderClient, cleanupSeq := setupExpressLaneAuction(t, tmpDir, ctx, jwtSecretPath)
+	defer cleanupSeq()
+	chainId, err := seqClient.ChainID(ctx)
+	Require(t, err)
+
+	auctionContract, err := express_lane_auctiongen.NewExpressLaneAuction(auctionContractAddr, seqClient)
+	Require(t, err)
+	info, err := auctionContract.RoundTimingInfo(&bind.CallOpts{})
+	Require(t, err)
+
+	// Prepare clients that can submit txs to the sequencer via the express lane.
+	seqDial, err := rpc.Dial(seq.Stack.HTTPEndpoint())
+	Require(t, err)
+	createExpressLaneClient := func(name string) (*expressLaneClient, bind.TransactOpts) {
+		priv := seqInfo.Accounts[name].PrivateKey
+		expressLaneClient := newExpressLaneClient(
+			priv,
+			chainId,
+			time.Unix(info.OffsetTimestamp, 0),
+			arbmath.SaturatingCast[time.Duration](info.RoundDurationSeconds)*time.Second,
+			auctionContractAddr,
+			seqDial,
+		)
+		expressLaneClient.Start(ctx)
+		transacOpts := seqInfo.GetDefaultTransactOpts(name, ctx)
+		transacOpts.NoSend = true
+		return expressLaneClient, transacOpts
+	}
+	bobExpressLaneClient, bobOpts := createExpressLaneClient("Bob")
+	aliceExpressLaneClient, aliceOpts := createExpressLaneClient("Alice")
+
+	// Transfer express lane control from Bob to Alice
+	roundDuration := time.Minute
+	currRound := timeboost.CurrentRound(time.Unix(info.OffsetTimestamp, 0), roundDuration)
+	fmt.Println("asdfasdff ", currRound, err)
+
+	duringRoundTransferTx, err := auctionContract.ExpressLaneAuctionTransactor.TransferExpressLaneController(&bobOpts, currRound, seqInfo.Accounts["Alice"].Address)
+	Require(t, err)
+	err = bobExpressLaneClient.SendTransaction(ctx, duringRoundTransferTx)
+	Require(t, err)
```

Comment:
This is working because `setupExpressLaneAuction` does a lot of steps ending with Bob being the EL controller. Can this be refactored so `setupExpressLaneAuction` does less stuff and this test is more explicit about what's going on? Redoing everything in `setupExpressLaneAuction` also makes the test take longer.